app/containers/Facilities/FacilitiesListContainer.tsx

```diff
@@ -0,0 +1,143 @@
+import React, {useEffect} from 'react';
+import {graphql, createRefetchContainer} from 'react-relay';
+import {Table, Container, Row, Col} from 'react-bootstrap';
+import SortableTableHeader from 'components/SortableTableHeader';
+import SearchBox from 'components/SearchBox';
+import FacilitiesRowItemContainer from './FacilitiesRowItemContainer';
+
+export const FacilitiesList = props => {
+  const {
+    searchDisplay,
+    direction,
+    orderByField,
+    searchField,
+    searchValue,
+    handleEvent
+  } = props;
+  const {edges} = props.query.searchAllFacilities;
+  console.log(edges);
+  useEffect(() => {
+    const refetchVariables = {
+      searchField,
+      searchValue,
+      orderByField,
+      direction
+    };
+    props.relay.refetch(refetchVariables);
+  });
+
+  const tableHeaders = [
+    {columnName: 'organisation_name', displayName: 'Organisation Name'},
+    {columnName: 'facility_name', displayName: 'Facility Name'},
+    {columnName: 'facility_mailing_address', displayName: 'Address'},
+    {columnName: 'facility_postal_code', displayName: 'Postal Code'},
+    {columnName: 'facility_city', displayName: 'City'},
+    {columnName: 'application_status', displayName: 'Status'}
+  ];
+
+  const dropdownSortItems = [
+    'Organisation Name',
+    'Facility Name',
+    'Address',
+    'Postal Code',
+    'City',
+    'Status'
+  ];
+
+  const displayNameToColumnNameMap = {
+    'Organisation Name': 'organisation_name',
+    'Facility Name': 'facility_name',
+    Address: 'facility_mailing_address',
+    'Postal Code': 'facility_postal_code',
+    City: 'facility_city',
+    Status: 'application_status'
+  };
+
+  return (
+    <>
+      <Container>
+        <Row>
+          <Col md={{span: 12, offset: 6}}>
+            <SearchBox
+              dropdownSortItems={dropdownSortItems}
+              handleEvent={handleEvent}
+              displayNameToColumnNameMap={displayNameToColumnNameMap}
+              searchDisplay={searchDisplay}
+            />
+          </Col>
+        </Row>
+      </Container>
+
+      <Table striped bordered hover style={{textAlign: 'center'}}>
+        <thead>
+          <tr>
+            {tableHeaders.map(header => (
+              <SortableTableHeader
+                key={header.columnName}
+                sort={handleEvent}
+                headerVariables={header}
+              />
+            ))}
+            <th />
+          </tr>
+        </thead>
+        {/* filter will not work if there's duplicate facilities in the table, bc the key is not unique */}
```

Comment:
Does this still apply? The key needs to be unique, otherwise React may be very confused

Reply:
this has been fixed